server/area.py

```diff
@@ -1720,9 +1720,21 @@ def change_status(self, value):
             "recess",
             "gaming",
         )
-        if value.lower() not in allowed_values:
+        value = censor(
+            value,
+            self.server.censors["whole"],
+            self.server.censors["replace"],
+            True,
+        )
+        value = censor(
+            value,
+            self.server.censors["partial"],
+            self.server.censors["replace"],
+            False,
+        )
```

Comment:
why is this copy-pasted twice

Reply:
the first one censors["whole"] the second one is censors["partial"] I saw it in AO protocol file